source/time_tables/passages.cpp

```diff
@@ -191,7 +191,8 @@ void passages(PbCreator& pb_creator,
 
         const type::VehicleJourney* vj = dt_stop_time.second->vehicle_journey;
         passage->mutable_stop_date_time()->set_data_freshness(to_pb_realtime_level(vj->realtime_level));
-        const auto& vj_st = navitia::VjStopTimes(vj, dt_stop_time.second, nullptr);
+        const auto sts = std::vector<const nt::StopTime*>{dt_stop_time.second};
+        const auto& vj_st = navitia::VjStopTimes(vj, sts);
```

Comment:
i don't understand why you did a copy at L194? since it's declared `const`, why don't you pass the ref of `dt_stop_time.second` to `navitia::VjStopTimes` directly?